.github/workflows/test-lint.yml

```diff
@@ -31,7 +32,7 @@ jobs:
       - uses: golangci/golangci-lint-action@v3
         if: env.GIT_DIFF
         with:
-          version: v1.54.2
+          version: v1.52.1
```

Comment:
Why is it downgraded?

Reply:
Maybe #3700 might add some context, I think the reason is related to it